lib/active_model/serializer.rb

```diff
@@ -112,10 +112,25 @@ def self.digest_caller_file(caller_line)
       Digest::MD5.hexdigest(serializer_file_contents)
     end
 
+    # TODO(beauby): Cache this.
+    def self.serializer_lookup_chain_for(klass)
```

Comment:
me too, but I would like to prefer to over document than to under document. Like, look at how rails  documents everything with plenty of examples: https://github.com/rails/rails/blob/master/activemodel/lib/active_model/errors.rb